LondonBritishLibrary/orient/BLorient689.xml

```diff
@@ -0,0 +1,493 @@
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+   schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="BLorient689" type="mss">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Gadla samāʿtāt</title>
+                <editor key="DE"></editor>
+                <editor key="AB" role="generalEditor"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 689</idno>
+                        <altIdentifier>
+                            <idno>Wright cat. CCLIII</idno>
+                        </altIdentifier>
+                        <altIdentifier>
+                            <idno>Wright 253</idno>
+                        </altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                          <locus from="1r"></locus>
+                          <title type="complete" ref="LIT1493Gadlas"></title>
+                          <textLang mainLang="gez"></textLang>
+                          <msItem xml:id="ms_i1.1">
+                            <locus from="1ra" to="7va"></locus>
+                            <title type="complete" ref="LIT2085OntheL"></title>
+                            <textLang mainLang="gez"></textLang>
+                            <incipit xml:lang="gez">
+                              <hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡</hi>
+                              ወመንፈስ፡ ቅዱስ፡ አሐ<hi rend="rubric">ዱ፡ አምላክ፡ ዝንቱ፡ ነ</hi>ገር፡
+                              ዘከመ፡ ኮነ፡ ሕይወ<hi rend="rubric">ቱ፡ ለቅዱስ፡ ዮሐንስ፡ በ</hi>ዕለ፡
+                              ክርስቲና፡ መጥምቅ፡ <hi rend="rubric">ወዘከመ፡ እፎ፡ ኮነ፡ ም</hi>ትረተ፡
+                              ርእሱ፡ ወተርጐሞ፡ <hi rend="rubric">ቅዱስ፡ ዮሐንስ፡ ልሳነ፡</hi> ወርቅ፡
+                              በትርያከ፡
+                            </incipit>
+                            <note>1st Maskaram</note>
```

Comment:
Could you mark up these dates with  `<date ref="ethiocal:Maskaram1">` etc.? I know it's not necessary, but I think it's a case where it's not too complicated and it will add some nice dates for future searches

Reply:
Thank you! I have forgotten this option.